stable/kubernetes-dashboard/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Delete the YAML separator and wrap the entire manifest in a `{{- if .Values.rbac.create }}...{{- end }}`